src/filter/EncodeFilter.cpp

```diff
@@ -52,7 +52,11 @@ class AudioEncodeFilterPrivate Q_DECL_FINAL : public AudioFilterPrivate
 AudioEncodeFilter::AudioEncodeFilter(QObject *parent)
     : AudioFilter(*new AudioEncodeFilterPrivate(), parent)
 {
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
     connect(this, SIGNAL(requestToEncode(QtAV::AudioFrame)), this, SLOT(encode(QtAV::AudioFrame)));
+#else
+    connect(this, &AudioEncodeFilter::requestToEncode, this, &AudioEncodeFilter::encode);
```

Comment:
I don't know!  it complains about not knowing what an AudioFrame is (even though that meta type was registered by QtAV) -- I actually tried calling qRegisterMetaTypes myself to explicitly register it, etc.  

Anyway I think the reason was you initially did the connect on "AudioFrame" rather than "QtAV::AudioFrame" -- but I got nervous -- runtime type checking and all is nice for flexibility but scary for testing.